samples/tutorials/mnist/README.md

```diff
@@ -0,0 +1,110 @@
+for data preprocessing, data transformation, model training, and so on. 
+
+## Content Overview:
+In this tutorial, we designed a series of notebooks to demonstrate how to interact with `Kubeflow Pipelines` through
+[Python SDK](https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp). In particular
+- [00 Kubeflow Cluster Setup](00_Kubeflow_Cluster_Setup.ipynb): this notebook helps you deploy a Kubeflow 
+cluster through CLI. The [UI](https://www.kubeflow.org/docs/gke/deploy/deploy-ui/) method of deploying a Kubeflow 
+cluster does not support Kubeflow v0.7 yet.
+
+Then, notebooks 01-04 use one concrete use case, i.e., 
+[MNIST classification](https://www.tensorflow.org/tutorials/quickstart/beginner), to demonstrate different ways of
+authoring a pipeline component: 
+- [01 Lightweight Python Components](01_Lightweight_Python_Components.ipynb): this notebook demonstrates how to build a 
+component through defining a stand-alone python function and then calling `kfp.components.func_to_container_op(func)` to 
+convert, which can be used in a pipeline.
+
+- [02 Local Development with Docker Image Components](02_Local_Development_with_Docker_Image_Components.ipynb): this 
+notebook guides you on creating a pipeline component with `kfp.components.ContainerOp` from an existing Docker image 
+which should contain the program to perform the task required in a particular step of your ML workflow.
+
+- [03 Reusable Components](03_Reusable_Components.ipynb): this notebook describes the manual way of writing a full 
+component program (in any language) and a component definition for it. Below is a summary of the steps involved in 
+creating and using a component.
+    - Write the program that contains your component’s logic. The program must use files and command-line arguments 
+    to pass data to and from the component.
+    - Containerize the program.
+    - Write a component specification in YAML format that describes the component for the Kubeflow Pipelines system.
+    - Use the Kubeflow Pipelines SDK to load your component, use it in a pipeline and run that pipeline.
+
+- [04 Reusable and Pre-build Components as Pipeline](04_Reusable_and_Pre-build_Components_as_Pipeline.ipynb): this 
+notebook combines our built components, together with a pre-build GCP AI Platform components 
+and a lightweight component to compose a pipeline with three steps.
+    - Train a MINIST model and export it to GCS
+    - Deploy the exported Tensorflow model on AI Platform prediction service
+    - Test the deployment by calling the end point with test data
+
+## Setups Overview:
+### Prerequisites
+Before you follow the instructions below to deploy your own Kubeflow cluster, you should
+
+- have a [GCP project setup](https://www.kubeflow.org/docs/gke/deploy/project-setup/) for your Kubeflow deployment 
+with you having the [owner role](https://cloud.google.com/iam/docs/understanding-roles#primitive_role_definitions) 
```

Comment:
Owner role is sufficient but not necessary, correct? 
If so, could you adjust the role as the minimum required one?

Reply:
Hey, we actually cite from [here](https://www.kubeflow.org/docs/gke/deploy/project-setup/), that owner is the required.

We have also experimented, it seems owner is necessary.